azure-pipelines-official.yml

```diff
@@ -95,6 +95,7 @@ extends:
   parameters:
     featureFlags:
       autoBaseline: true
+      useArtifactDropTaskV1: true
```

Comment:
I added this simply because I converted `- task: 1ES.PublishArtifactsDrop@1` into `- output: artifactsDrop` and the 1ES docs say that without `useArtifactDropTaskV1: true`, the output would use `@0` version of the task.